client/src/pages/ManageBookings/ManageBookings.tsx

```diff
@@ -0,0 +1,141 @@
+import { useState } from 'react';
+import PageContainer from '../../components/PageContainer/PageContainer';
+import clsx from 'clsx';
+import { Box, Grid, Typography, Avatar } from '@mui/material';
+import useStyles from './useStyles';
+import CardWrapper from '../../components/CardWrapper/CardWrapper';
+import SettingsIcon from '@mui/icons-material/Settings';
+import { NavLink } from 'react-router-dom';
+import dogPicture from '../../images/landing/hero.jpg';
+
+import MyCalendar from '../../components/MyCalendar/MyCalendar';
+
+const RequestInfo: React.FC<{
+  date?: string | Date;
+  avatar?: any;
+  name?: string;
+  status?: string;
+  size?: any;
+}> = ({ date, avatar, name, status, size }) => {
+  const classes = useStyles();
+
+  return (
+    <Box className={clsx(classes.requestInfo, size === 'large' && classes.requestInfoLarge)}>
+      <Grid xs={9} item>
+        <Typography variant={size === 'large' ? 'h6' : 'body1'} gutterBottom>
+          {date}
+        </Typography>
+        <Box className={classes.nameAvatar}>
+          <Avatar
+            alt="Profile Image"
+            src={avatar}
+            sx={size === 'large' ? { width: 50, height: 50 } : { width: 40, height: 40 }}
```

Comment:
Also, wouldn't it be better to use breakpoints to apply styles for large/small screens accordingly?